packages/components/pie-card/src/card.scss

```diff
@@ -39,19 +39,19 @@
     outline: none;
     text-decoration: none;
 
-    &[variant='default'] {
+    &--default {
```

Comment:
I think @ashleynolan has had strong feelings about doing this in the past because it makes it harder to search the codebase for the class, and I think I agree.

Reply:
Completely forgot that - we should deffo make a lint rule at some point! Will move it out into its own selector